modular_zubbers/code/modules/clothing/head/helmet.dm

```diff
@@ -62,3 +62,69 @@
 	worn_icon = 'modular_zubbers/icons/mob/clothing/head/hats.dmi'
 	icon_state = "clussy_wig"
 	flags_inv = HIDEHAIR
+
+/obj/item/clothing/head/playbunnyears
+	name = "bunny ears headband"
+	desc = "A pair of bunny ears attached to a headband. One of the ears is already crooked."
+	icon = 'modular_zubbers/icons/obj/clothing/head/hats.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/head/bunnyears.dmi'
+	icon_state = "playbunny_ears"
+	greyscale_colors = "#39393f"
+	greyscale_config = /datum/greyscale_config/playbunnyears
+	greyscale_config_worn = /datum/greyscale_config/playbunnyears_worn
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/head/playbunnyears/syndicate
+	name = "blood-red bunny ears headband"
+	desc = "An unusually suspicious pair of bunny ears attached to a headband. The headband looks reinforced with plasteel... but why?"
+	icon_state = "syndibunny_ears"
+	clothing_flags = SNUG_FIT
+	armor_type = /datum/armor/playbunnyears_syndicate
+	greyscale_colors = null
+	greyscale_config = null
+	greyscale_config_worn = null
+
+/datum/armor/playbunnyears_syndicate
+	melee = 30
+	bullet = 20
+	laser = 30
+	energy = 35
+	fire = 20
+	bomb = 15
+	acid = 50
+	wound = 5
```

Comment:
I think `/datum/armor/head_helmet` can work here

Reply:
No problemo. On it.